homeassistant/components/zwave/__init__.py

```diff
@@ -628,14 +627,7 @@ def __init__(self, hass, schema, primary_value, zwave_config,
         self._entity = None
         self._workaround_ignore = False
 
-        # Combine node value schemas and instance value schemas into one
-        # values schema mapping.
         self._schema[const.DISC_VALUES] = {}
-        for name in self._schema[const.DISC_NODE_VALUES].keys():
-            self._values[name] = None
-            self._schema[const.DISC_VALUES][name] = \
-                self._schema[const.DISC_NODE_VALUES][name]
-
         for name in self._schema[const.DISC_INSTANCE_VALUES].keys():
```

Comment:
At this point we should just drop the `const.DISC_INSTANCE_VALUES` name and use `const.DISC_VALUES`, since they're all instance values now, and any node values in the future would always go to the node entity.